test/watchers/payload_tracker_watcher_test.py

```diff
@@ -59,8 +59,7 @@ def test_payload_tracker_init_with_kafka_config(producer_init_mock):
     """Check that passing a kafka_broker_config parameter updates the default ones."""
     kafka_broker_cfg = {"bootstrap.servers": "valid_server"}
 
-    producer_mock = _prepare_kafka_mock(producer_init_mock)
```

Comment:
is this removed in purpose?

Reply:
Yes, I see some lintian warnings locally and I removed it to avoid CI issues